### PR TITLE
alerting/webhook doc update

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

updates current webhook documentation https://grafana.com/docs/grafana/next/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier/
moves wecom integration into a separate doc

### CODE REVIEW

**Comment by tonypowa** [comment]
testing with `make docs`

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md [hunk 1]
```diff
@@ -24,9 +24,11 @@ title: Configure the webhook notifier for Alerting
 weight: 200
 ---
 
-### Configure the webhook notifier for Alerting
+## Configure the webhook notifier for Alerting
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md:27]: this should be header 1 #

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md [hunk 2]
```diff
@@ -24,9 +24,11 @@ title: Configure the webhook notifier for Alerting
...
 
-Example JSON body:
+The webhook notification is a simple way to send information about a state change over HTTP to a custom endpoint. Using this notification you could integrate Grafana into a system of your choosing.
+
+### Webhook JSON payload
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md:31]: ##

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md (hunk 1 above)
Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md:27]: oh then we did not get it right in the skack and telegram docs 😁

**Review by brendamuir — APPROVED** [review]

**Review by brendamuir — APPROVED** [review]

**Pull request merged**